tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -106,6 +106,8 @@ readonly -a systemd_files=(
 	"systemd-tmpfiles-cleanup.timer"
 )
 
+typeset delete_unnecessary_files="yes"
```

Comment:
Can we rename this so it doesn't conflict with the function of the same name?
```suggestion
typeset should_delete_unnecessary_files="yes"
```